_posts/2019-01-30-gcal-for-project-management.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Using Google Calendar for Account Management"
+date: 2019-01-30 10:00:00
+categories: ["agile", "project-management"]
+author: "emily"
+---
+
+At [Ombu Labs](https://www.ombulabs.com), we like to split our time between working on our own products, open source code and client projects.
```

Comment:
@ercohen14 It might sound better like this: 

, open source and client projects

or: 

, open source code, and client projects.